argo-cd-apps/base/member/infra-deployments/mintmaker/mintmaker.yaml

```diff
@@ -17,6 +17,10 @@ spec:
               elements:
                 - nameNormalized: stone-stage-p01
                   values.clusterDir: stone-stage-p01
+                - nameNormalized: stone-prod-p01
+                  values.clusterDir: stone-prod-p01
```

Comment:
Seems p01 will be decommissioned, should we remove this?

Reply:
let it there for now. when decommissioned, someone will clean this up.